apps/backend/src/core/application/repositories/user-session.repository.ts

```diff
@@ -4,5 +4,7 @@ export const USER_SESSION_REPOSITORY = 'USER_SESSION_REPOSITORY';
 
 export default interface UserSessionRepository {
   getByUserId: (userId: string) => Promise<UserSession>;
-  store: (userSession: UserSession) => Promise<void>;
+  getById: (id: string) => Promise<UserSession>;
+  store: (userSession: UserSession) => Promise<UserSession['id']>;
+  delete: (id: string) => Promise<void>;
```

Comment:
Is it possible to be more explicit about the output shape of this Promise?

```suggestion
  delete: (id: string) => Promise<something_else>;
```